### ISSUE TITLE
Test new version of Salmon without enabling mapping validation

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#1287]
### Context

Related to: #1101, #1159 

We are using `v0.9.1` of Salmon. That means there are several versions worth of bug fixes that we do not take advantage of. It is unclear how long the research and implementation of the trimming portion described in #1159 will take at this time and the trimming is most important for using mapping validation. 

It may behoove us to use a newer Salmon version without taking advantage of this new mode, as it could mean more successful RNA-seq jobs and gets around the trimming as a potential blocker.

### Problem or idea

1. Upgrade Salmon to the current version ([`v0.13.1`](https://github.com/COMBINE-lab/salmon/releases/tag/v0.13.1))
2. Generate new transcriptome indices for _Danio rerio_ on staging
3. Run the RNA-seq pipeline using `salmon quant` without changing the flags for a random subset of zebrafish experiments
4. Compare these new values (output of tximport) to the values we have on prod

### Solution or next step

I'm generating list of random experiments now

**Comment by jaclyn-taroni** [comment]
@kurtwheeler here is a list of 20 randomly selected zebrafish RNA-seq experiments: [random_zebrafish_rnaseq_projects.txt](https://github.com/AlexsLemonade/refinebio/files/3132019/random_zebrafish_rnaseq_projects.txt)

This accounts for ~400 samples and the experiments range in size.

**Comment by cansav09** [comment]
Plan is that after steps 1-2 have been finished, @kurtwheeler will complete step 3 by getting me `0.13.1` and `0.9.1` Salmon versions of `LengthScaledTPM` files for each sample that is in the datasets in `random_zebrafish_rnaseq_projects.txt` list. He will have each file labeled with its sample ID (`SRR`) and it's dataset ID (`SRP`) and it's Salmon version (`0.13.1` or `0.9.1`), so that I can have the info I need for each sample.

Then I will do step 4 by performing Spearman's correlations and RMSE's between the versions, and matching each sample. Depending on how that turns out we will figure out what further tests need to be done. 

The output results I will probably make into a jitterplot/boxplot for each experiment and maybe a density plot with all the samples combined.

**Comment by kurtwheeler** [comment]
OK so @arielsvn put our heads together and decided that despite the skip_qn feature being implemented on the frontend we can do a little magic to enable that feature for these datasets in particular.

So basically I'll use the frontend to generate the datasets and have them sent to your email.

**Comment by kurtwheeler** [comment]
Ok so it looks like a number of the experiments we chose have been blacklisted by SRA. This is a list of what did and didn't work in staging:
```
SRP050879 -- yes
SRP028895 -- no
SRP040018 -- yes
SRP105338 -- no
SRP074122 -- no
SRP101561 -- yes
SRP072957 -- no
SRP093723 -- no
SRP065358 -- yes
SRP075408 -- yes
SRP042975 -- no
ERP007082 -- no
ERP005517 -- no
SRP043065 -- yes
SRP024369 -- yes
ERP013690 -- yes
SRP087617 -- no
SRP117794 -- yes
SRP034166 -- yes
SRP072298 -- no
```

We talked with @jaclyn-taroni and we decided that we'll run the analysis as expected on the experiments that we were able to reprocess. If things look good, then we're done. If things look bad then we'll find additional experiments that we can reprocess to determine how bad.

So, I am going to send another dataset @cansav09's way from staging with the experiments that did work.
